nifi-nar-bundles/nifi-standard-services/nifi-elasticsearch-client-service-bundle/nifi-elasticsearch-client-service-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,332 @@
+nifi-hbase_1_1_2-client-service-nar
```

Comment:
This appears to be a copy from a different NAR ;) The first line should be updated, and the rest of the NOTICES below should only be included if we are including the component(s) in the artifacts.